build-managed.sh

```diff
@@ -1,5 +0,0 @@
-#!/usr/bin/env bash
-
-__scriptpath=$(cd "$(dirname "$0")"; pwd -P)
-$__scriptpath/run.sh build-managed $@
-exit $?
```

Comment:
how come we didn't also delete build-managed.cmd?

Reply:
There isn't a build-managed.cmd